### PR TITLE
fixing esc in evtol weight buildup

### PR DESCRIPTION
**Pull request by mclarke2**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py [hunk 1]
```diff
@@ -140,12 +140,11 @@ def empty(config,
     #-------------------------------------------------------------------------------    
     MTOW                = config.mass_properties.max_takeoff  
     output.seats        = config.passengers * 15.   * Units.kg
-    output.passengers   = config.passengers * 70.   * Units.kg
+    output.passengers   = config.passengers * 62.   * Units.kg
```
Comment (planes) [inline trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py:143]: I know it makes your results look really good. But we're not designing around children.

https://www.faa.gov/documentlibrary/media/advisory_circular/ac120-27e.pdf

If you want 62 kg adults then make a default to a realistic value and change your own scripts.

File: trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py [hunk 2]
```diff
@@ -192,21 +197,6 @@ def empty(config,
         propulsor.avionics.origin[0][0]                                      = 0.4 * nose_length
         propulsor.avionics.mass_properties.center_of_gravity[0][0]           = 0.0
         propulsor.avionics.mass_properties.mass                              = output.avionics
-        
-        #-------------------------------------------------------------------------------
```
Comment (planes) [inline trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py:196]: Wait wait, did you delete the electronic speed controller weight for the environmental control system?

Comment (mclarke2) [reply trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py:196]: I made a mistake - First I assigned ECS to ESC. then when I corrected it Jordan told me that it was in fact supposed to be ECS - environmental control system. There was never a weight build-up for ESC after all. This is of course why we always stress comments to escape this confusion.

File: trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py (hunk 1 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py:143]: Oh I will change it back.

**Review by planes — APPROVED** [review]

**Pull request merged**